modal_proto/api.proto

```diff
@@ -2378,6 +2387,9 @@ service ModalClient {
   rpc VolumeReload(VolumeReloadRequest) returns (google.protobuf.Empty);
   rpc VolumeRemoveFile(VolumeRemoveFileRequest) returns (google.protobuf.Empty);
 
+  // Web
+  rpc WebDashboard(WebDashboardRequest) returns (WebDashboardResponse);
```

Comment:
All other getter RPCs use `GetOrCreate` (`VolumeGetOrCreate`, `SharedVolumeGetOrCreate`). We are not creating a dashboard in this case but we are getting it. What about using `WebDashboardGet` instead?

Reply:
Another existing pattern is "Lookup", so maybe `WebDashboardLookup`? That might fit better with returning a pointer to it (although I guess the `Get` RPCs really just return IDs anyway).

Reply:
Yeah I like it! Updated this to `WebDashboardLookup` now